src/_guides/language/effective-dart/usage.md

```diff
@@ -21,6 +21,106 @@ library may not be able to tell that you've internalized the ideas here, but
 * TOC
 {:toc}
 
+## Libraries
+
+These guidelines help you compose your program out of multiple files in a
+consistent, maintainable way. To keep these guidelines brief, they use "import"
```

Comment:
Maybe: they use "import" --> they use `import`. @kwalrath WDYT?

Reply:
I think in this case "import" is what we mean, because we're talking about text, not keywords.

Reply:
(I personally would use _italics_ instead of quotation marks, but I can't find a style guide ruling either way.)